src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -245,36 +245,38 @@ def test_commit(single_channel_session):
 def test_configure_digital_edge_measure_trigger(single_channel_session):
     single_channel_session.measure_when = nidcpower.MeasureWhen.ON_MEASURE_TRIGGER
     expected_trigger_terminal = "/4162/PXI_Trig0"
-    single_channel_session.configure_digital_edge_measure_trigger(expected_trigger_terminal)
+    single_channel_session.digital_edge_measure_trigger_input_terminal = expected_trigger_terminal
```

Comment:
We didn't add all attributes to our system tests. We had added the functions as a way to validate the function metadata.

Seems like we should be consistent and rip these out as well.

Reply:
Removed all tests that became attribute only.